src/App.tsx

```diff
@@ -1,14 +1,80 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { Filter } from './Filter';
 
 export const App: React.FC = () => {
+  const [isLoadedTodos, setIsLoadedTodos] = useState(false);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [isOpenedTodo, setIsOpenedTodo] = useState(false);
+  const [selectFilter, setSelectFilter] = useState('all');
+  const [searchQuery, setSearchQuery] = useState('');
+
+  useEffect(() => {
+    const loadTodos = async () => {
+      await getTodos().then(todoList => {
+        setTodos(todoList);
+        setVisibleTodos(todoList);
+      });
+
+      setIsLoadedTodos(true);
+    };
+
+    loadTodos();
+  }, []);
```

Comment:
use either `.then()`, `.catch`, `.finally()` or `async`/`await`, but not both
```suggestion
  useEffect(() => {
    getTodos().then(todoList => {
      setTodos(todoList);
      setVisibleTodos(todoList);
      setIsLoadedTodos(true);
    });
  }, []);
```